src/sites/image/imgsee.me.js

```diff
@@ -253,21 +80,6 @@
     });
   }
 
-  // Used when the form's shell does not exist when page loaded.
-  async function getAmbiguousForm (selector, shellNormalizer) {
-    const d = await waitFormShell(selector, shellNormalizer);
-    const style = $('style', d);
-    let visibleClasses = parseStyle(style);
-    visibleClasses = filterDuplicated(visibleClasses);
-    while (true) {
-      const button = findVisibleForm(visibleClasses);
-      if (button) {
-        return button;
-      }
-      await _.wait(500);
-    }
-  }
-
   // Used when the form's shell does not exist when page loaded.
   // Wait for the given selector, and normalize matched element by normalizer.
   function waitFormShell (selector, normalizer) {
```

Comment:
with your code changes... you can delete this function.
github indicates:   _'waitFormShell' is defined but never used_

Reply:
also
_'parseStyle' is defined but never used_   
_'filterDuplicated' is defined but never used_
_'findVisibleForm' is defined but never used_

Reply:
I tried removing those sections and tested it in my fork. It still failed with a generic 
"Process completed with exit code 1."

https://github.com/Reggie521/adsbypasser/actions/runs/6700240405

Reply:
ok, looking at the build run...
this is the reason for the failure...: it was on line 126
_126:1  error  Parsing error: Unexpected token (126:1)_

![image](https://github.com/adsbypasser/adsbypasser/assets/13878580/88864bb8-db7a-49d2-ac8b-1b6bf42579fd)